python/ribasim/ribasim/geometry/node.py

```diff
@@ -59,6 +63,34 @@ def node_ids_and_types(*nodes):
 
         return node_id, node_type
 
+    def translate_spacially(
```

Comment:
Both functions here are identical to the one in edge.py, try to generalize these.